index.html

```diff
@@ -25,6 +22,9 @@ <h4>Great Sightings</h4>
       <!-- add an <li> for each animal sighting here -->
     </ul>
   </div>
+
+  <script src="data.js"></script>
+  <script src="index.js"></script>
```

Comment:
Instead of doing this. Let's put these script tags back in the head tag of th HTML doc and then add a defer to the index.js script tag
 <script src="index.js" defer></script>